django/dkg/cancer/model_api/model.py

```diff
@@ -0,0 +1,36 @@
+import numpy
+
+
+def train_articles_adapter(model_articles):
```

Comment:
glaub ich hab die struktur verstanden, aber mir is noch nich klar, wo die zu labeldn artikel priorisiert werden. wollen wir das in train_model machen und die vorhersagen, mit diversen priorisierungsheuristiken gleich in ne table schreiben?